src/main/java/org/apache/sling/distribution/serialization/impl/vlt/FileVaultContentSerializer.java

```diff
@@ -131,7 +133,7 @@ public void importFromStream(ResourceResolver resourceResolver, InputStream inpu
         Archive archive = null;
         try {
             session = getSession(resourceResolver);
-            ImportOptions importOptions = VltUtils.getImportOptions(aclHandling, cugHandling, importMode, autosaveThreshold, strict);
+            ImportOptions importOptions = VltUtils.getImportOptions(aclHandling, cugHandling, importMode, autosaveThreshold, strict, idConflictPolicy);
             ErrorListener errorListener = new ErrorListener();
             importOptions.setListener(errorListener);
             Importer importer = new Importer(importOptions);
```

Comment:
what about the other defaults: 

- boolean isStrictByDefault
- boolean overwritePrimaryTypesOfFoldersByDefault

You should either inherit form the Packaging OSGi configuration or make them maintainable as well IMHO.

Reply:
@kwin We already have a property for strict mode - [here](https://github.com/apache/sling-org-apache-sling-distribution-core/blob/501c4e8824458454e676e8c62a7398b68f309a98/src/main/java/org/apache/sling/distribution/serialization/impl/vlt/VaultDistributionPackageBuilderFactory.java#L168).

I wasn't sure if we needed `overwritePrimaryTypesOfFoldersByDefault`, please let me know if we should include that.

Reply:
Agreed with @kwin point. We'd need to reuse the existing configuration as a base.

@kwin can we get the configuration through an API ?

Reply:
We may also get the configuration by registering a ManagedServiceFactory like [here](https://github.com/apache/sling-org-apache-sling-distribution-journal/blob/787e79f07a4a165eda5d95a17aafe490f1aea906/src/main/java/org/apache/sling/distribution/journal/shared/PublisherConfigurationAvailable.java) but we'd still need the keys to be exported.

Reply:
There is no method you can use to access the high level configuration object from https://github.com/apache/jackrabbit-filevault/blob/ffe9984bd965b276f93eb776374f5dde63cb0a30/vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/impl/PackagingImpl.java#L116, but you can use the OSGi configuration admin to get the keys and values.

Reply:
Thanks @kwin. Agreed, we can either use the ConfigurationAdmin, or a NabagedServiceFactory. But my point is, we'd need the key exported as constants to make it clean (not redefine the constants in our code base).

Reply:
@ManasMaji IMO we should obtain the default from Packaging configuration by implementing a ManagedServiceFactory, and override the defaults the same way we do today.

Reply:
@tmaret Feel free to raise a PR for FileVault.

Reply:
@kwin I have tracked https://issues.apache.org/jira/browse/JCRVLT-656 to export the configuration.

@ManasMaji Since it'll need a FileVault release, I think we could duplicate the constants in Sling Content Distribution until JCRVLT-656 is done.